tests/bsim/bluetooth/host/iso/frag/src/broadcaster.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/iso.h>
+#include <zephyr/logging/log.h>
+
+#include "babblekit/flags.h"
+#include "babblekit/testcase.h"
+
+LOG_MODULE_REGISTER(broadcaster, LOG_LEVEL_INF);
+
+static struct bt_iso_chan iso_chans[CONFIG_BT_ISO_MAX_CHAN];
+static struct bt_iso_chan *default_chan = &iso_chans[0];
+
+NET_BUF_POOL_FIXED_DEFINE(tx_pool, CONFIG_BT_ISO_TX_BUF_COUNT,
+			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
+			  CONFIG_BT_CONN_TX_USER_DATA_SIZE, NULL);
+
+static DEFINE_FLAG(iso_connected);
+static DEFINE_FLAG(sdu_sent);
+
+static int send_data(struct bt_iso_chan *chan, bool ts)
+{
+	static uint16_t seq;
+	struct net_buf *buf;
+	int err;
+
+	if (!IS_FLAG_SET(iso_connected)) {
+		/* TX has been aborted */
+		return -ENOTCONN;
+	}
+
+	buf = net_buf_alloc(&tx_pool, K_NO_WAIT);
+	TEST_ASSERT(buf != NULL, "Failed to allocate buffer");
+
+	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
+
+	net_buf_add_le32(buf, 0xdeadbeef);
+	net_buf_add_u8(buf, 0x11);
+
+	LOG_INF("Sending SDU with%s timestamp (headroom %d)", ts ? "" : "out", net_buf_headroom(buf));
```

Comment:
You can run locally 
`scripts/ci/check_compliance.py -c main..HEAD`

Reply:
Yeah I know. I only do it if I have one failure.